oodeel/utils/tf_operator.py

```diff
@@ -57,6 +73,33 @@ def sign(tensor: Union[tf.Tensor, np.ndarray]) -> tf.Tensor:
         return tf.sign(tensor)
 
     @staticmethod
+    def CrossEntropyLoss(reduction: str = "mean"):
```

Comment:
`reduction="mean"` is not a supported argument for Keras losses. Maybe do you mean `sum_over_batch_size`? Or even `auto`?